artifacts/crds/inlets.inlets.dev_tunnels.yaml

```diff
@@ -86,4 +86,4 @@ status:
     kind: ""
     plural: ""
   conditions: null
-  storedVersions: null
+  storedVersions: []
```

Comment:
Did you get this change through a manual edit or by running the script in hack/ to update the CRD?

Reply:
I changed it manually. I didn't know about this hack/ script. I will re-generate this using hack/.

Reply:
The issue with the manual edit is that the script will overwrite it, so I don't mind changes if they are generated, otherwise we'll lose them.

Should be -> `/hack/update-crd.sh`

Reply:
I tried `/hack/update-crd.sh`. The `controller-tools` in the script is not generate expected `storedVersions: []`.

It seems that some versions of controller-tools generate `storedVersions: []`, but some others generate `storedVersions: null`.

I tried the master branch of controller-tools, still cannot generate `storedVersions: []`. But someone did fix the similar problem by upgrading controller-tools: https://github.com/pusher/faros/issues/75